src/sentry/tasks/backfill_seer_grouping_records.py

```diff
@@ -0,0 +1,335 @@
+import logging
+import time
+from datetime import datetime, timedelta
+from typing import Any, TypedDict
+
+import sentry_sdk
+from django.conf import settings
+from google.api_core.exceptions import DeadlineExceeded, ServiceUnavailable
+from snuba_sdk import Column, Condition, Entity, Function, Op, Query, Request
+from snuba_sdk.orderby import Direction, OrderBy
+
+from sentry import features, nodestore
+from sentry.api.endpoints.group_similar_issues_embeddings import get_stacktrace_string
+from sentry.eventstore.models import Event
+from sentry.grouping.grouping_info import get_grouping_info
+from sentry.issues.grouptype import ErrorGroupType
+from sentry.issues.occurrence_consumer import EventLookupError
+from sentry.models.group import Group
+from sentry.models.grouphash import GroupHash
+from sentry.models.project import Project
+from sentry.seer.utils import (
+    CreateGroupingRecordData,
+    CreateGroupingRecordsRequest,
+    post_bulk_grouping_records,
+)
+from sentry.silo.base import SiloMode
+from sentry.snuba.dataset import Dataset
+from sentry.snuba.referrer import Referrer
+from sentry.tasks.base import instrumented_task
+from sentry.utils import json, metrics, redis
+from sentry.utils.iterators import chunked
+from sentry.utils.query import RangeQuerySetWrapper
+from sentry.utils.safe import get_path
+from sentry.utils.snuba import bulk_snuba_queries
+
+ITERATOR_CHUNK = 20
+SEER_BACKFILL_DELAY_PER_RECORD = 0.1
```

Comment:
let's consider making these options if in the future we want to quickly change these. but fine for now.